src/olympia/addons/models.py

```diff
@@ -1627,6 +1639,8 @@ class AddonReviewerFlags(ModelBase):
     needs_admin_content_review = models.BooleanField(default=False)
     needs_admin_theme_review = models.BooleanField(default=False)
     auto_approval_disabled = models.BooleanField(default=False)
+    auto_approval_disabled_until = models.DateTimeField(
```

Comment:
`auto_approval_delayed_until`? I think that would be more consistent.

Reply:
I like that. eefefa57c1e3851a0503e3fd3592829d0718335e